backend/logic/courses/courses.js

```diff
@@ -261,12 +261,108 @@ var updateCourse = async (course, newTitle, newTags) => {
   }
 };
 
+/**
+ * Assign a course to a curriculum
+ * @author Yufei Liu
+ * @param {string} curriculumType,{string} courseCode,{string} curriculum
```

Comment:
curriculumType should really be courseType because at first I thought you were referring to major vs minor or even 7 sem vs 8 sem curriculum.

Reply:
make sense, will change that